include/seqan3/argument_parser/validators.hpp

```diff
@@ -165,10 +166,15 @@ class arithmetic_range_validator
  *
  * \details
  *
- * On construction, the validator must receive a list (vector) of valid values.
+ * On construction, the validator must receive a range or parameter pack of valid values.
  * The struct than acts as a functor, that throws a seqan3::parser_invalid_argument
  * exception whenever a given value is not in the given list.
  *
+ * \note In order to simplify the chaining of validators, all ranges of arithmetic types resolve to the validator
+ *       value type `double` and all ranges of types which are convertible to resolve to the value type std::string.
+ *       Otherwise, e.g. `value_list_validator{"fooDir", "barDir"}` would resolve to value type `const char *`
```

Comment:
This is not what I meant. What if my types are `foo`, which are neither arithmetic nor convertible to string. That is still a valid validator, isn't it? At least there is nothing in the code that prevents that.

Reply:
ah ok. then please elaborate further next time :smile: 
yes if it is neither arithmetic nor convertible to std::string it is merely the type itself